packages/phone-number-privacy/combiner/jest.config.js

```diff
@@ -3,4 +3,5 @@ const { nodeFlakeTracking } = require('@celo/flake-tracker/src/jest/config.js')
 module.exports = {
   preset: 'ts-jest',
   ...nodeFlakeTracking,
+  setupFilesAfterEnv: ['<rootDir>/jest_setup.ts'],
```

Comment:
I know it's not pretty, but to keep the flake tracker enabled this line should be
```suggestion
  setupFilesAfterEnv: ['<rootDir>/jest_setup.ts', ...nodeFlakeTracking.setupFilesAfterEnv],
```

Reply:
Good catch, thanks!